README.md

```diff
@@ -41,6 +41,26 @@ The following bounties are available for anyone to pick up:
 
 <br>
 
+# Bounty Submission
+
+## [XRPLF](https://github.com/XRPLF) repos
+To submit your bounty work for [XRPLF](https://github.com/XRPLF) repos, please follow the corresponding repo's CONTRIBUTING.md's instructions to create a PR against the default branch. 
```

Comment:
These instructions don't just apply to XRPLF - They apply to any bounty which talks about an existing open source project. (Suggestion for generalizing the language)

```suggestion
## Contributing to existing open source projects
To submit your bounty work to an existing open source repository, (Ex. for contributions to [xrpl.org](https://github.com/XRPLF/xrpl-dev-portal) please follow the corresponding repo's CONTRIBUTING.md's instructions to create a pull request for their maintainers to review. 
```